src/cloudflare/internal/ai-api.ts

```diff
@@ -97,6 +96,17 @@ function isReadableStream(obj: unknown): obj is ReadableStream {
   );
 }
 
+function isFormData(obj: unknown): obj is FormData {
```

Comment:
Why not use something like this?

```
function isFormData(obj: unknown): obj is FormData {
  return obj instanceof FormData;
}
```

Reply:
It depends on what you're trying to support. Do you want to limit it only to the `globalThis.FormData` implementation provided by the runtime? If that's the case, then you'll want to use `instanceof globalThis.FormData` specifically. That's required because at the runtime internals level, `FormData` is backed by a c++ class and at the native/c++ level we will work with the C++ api and not the JS API.

However, if you want to be able to work with polyfills of the `FormData` API..  something that is implemented entirely in JavaScript, then you'll need to implement the adaptation code yourself and using `instanceof` is likely not what you want.